lib/DartEngine/ClassicGame.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "ModeGame.hpp"
+
+class ClassicGame : public ModeGame
```

Comment:
It better when it stay it that way because if we load a game from `GameSettings` we will need only a `GameStyle` and not more parametr to create game.

example:
```c++
Game::Game(GameSettings settings)
: Game(settings.style)
{
    mGameMode->setAttemps(settings.currentPlayerAttemps);
    mPlayers = settings.players;
    mWinners = settings.winners;
}
```

Reply:
now, I think it can be done, but I like it better in this form